src/startScreen.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './startScreen.css'
+
+import ARROW_KEYS_TUTORIAL from './assets/arrow-keys-tutorial.png';
+
+export default class StartScreen extends React.Component {
```

Comment:
Because this component doesn't really need it's own state we can define it as a functional component - let me know if you need help with that